app/client/cli/docgen/main.go

```diff
@@ -0,0 +1,17 @@
+package main
```

Comment:
Do we have a Makefile target to generate these?

Reply:
Yep: `make generate_cli_commands_docs`.